go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	flamingo.me/dingo v0.2.10
 	github.com/coreos/go-oidc/v3 v3.5.0
 	github.com/ghodss/yaml v1.0.0
-	github.com/go-redis/redis/v8 v8.11.5
+	github.com/go-redis/redis/v9 v9.0.0-rc.1
```

Comment:
Any reason why not to take the most recent version here `v9.0.3` ?

Reply:
The redis client we use changed their module path from `github.com/go-redis/redis` to `https://github.com/redis/go-redis` recently. `v9.0.0-rc.1` was the latest version that I was able to fetch with the old module path. 

Unfortunately, we can not yet switch to the new module path, because `github.com/rbcervilla/redisstore` also still uses the old path. If we switch to the new path, the compiler will complain about mismatching types when initializing the `redisstore`, because technically they are distinct packages.  I tried to resolve it with `replace` directives in the go.mod file, but I did not manage to get it working.

Reply:
`github.com/rbcervilla/redisstore` just switched to the new path, too. Now we can use the latest version of go-redis. I updated the PR already.